### PR TITLE
fix: исправление мелких ошибок

### PR DESCRIPTION
**Pull request by Kupov**

### CODE REVIEW

File: src/Tree/e-tree.scss [hunk 1]
```diff
@@ -5,19 +5,26 @@
...
+      }
     }
-    .e-tree-list .e-tree-list .e-tree-list .e-tree-item {
-      padding: 0 0 0 120px;
-      &:before { left: 100px; }
+    .e-tree-list .e-tree-list .e-tree-list {
+      .e-tree-item-title { max-width: 120px; }
+      .e-tree-item {
+        padding: 0 0 0 120px;
+        &:before { left: 100px; }
```
Comment (apress-hound) [inline src/Tree/e-tree.scss]: Selector should have depth of applicability no greater than 4, but was 5<br>Nesting should be no greater than 4, but was 5

File: src/Tree/e-tree.scss [hunk 2]
```diff
@@ -5,19 +5,26 @@
...
+        padding: 0 0 0 90px;
+        &:before { left: 70px; }
+      }
     }
-    .e-tree-list .e-tree-list .e-tree-list .e-tree-item {
-      padding: 0 0 0 120px;
-      &:before { left: 100px; }
+    .e-tree-list .e-tree-list .e-tree-list {
+      .e-tree-item-title { max-width: 120px; }
+      .e-tree-item {
```
Comment (apress-hound) [inline src/Tree/e-tree.scss]: Selector should have depth of applicability no greater than 4, but was 5

File: src/Tree/e-tree.scss [hunk 3]
```diff
@@ -5,19 +5,26 @@
...
+        padding: 0 0 0 90px;
+        &:before { left: 70px; }
+      }
     }
-    .e-tree-list .e-tree-list .e-tree-list .e-tree-item {
-      padding: 0 0 0 120px;
-      &:before { left: 100px; }
+    .e-tree-list .e-tree-list .e-tree-list {
+      .e-tree-item-title { max-width: 120px; }
```
Comment (apress-hound) [inline src/Tree/e-tree.scss]: Selector should have depth of applicability no greater than 4, but was 5

File: src/Tree/e-tree.scss [hunk 4]
```diff
@@ -5,19 +5,26 @@
...
+      .e-tree-item-title { max-width: 150px; }
+      .e-tree-item {
+        padding: 0 0 0 90px;
+        &:before { left: 70px; }
+      }
     }
-    .e-tree-list .e-tree-list .e-tree-list .e-tree-item {
-      padding: 0 0 0 120px;
-      &:before { left: 100px; }
+    .e-tree-list .e-tree-list .e-tree-list {
```
Comment (apress-hound) [inline src/Tree/e-tree.scss]: Merge rule `.e-tree-list .e-tree-list .e-tree-list` with rule on line 7

File: src/Tree/e-tree.scss [hunk 5]
```diff
@@ -5,19 +5,26 @@
...
       }
     }
-    .e-tree-list .e-tree-list .e-tree-item {
-      padding: 0 0 0 90px;
-      &:before { left: 70px; }
+    .e-tree-list .e-tree-list {
+      .e-tree-item-title { max-width: 150px; }
+      .e-tree-item {
+        padding: 0 0 0 90px;
+        &:before { left: 70px; }
```
Comment (apress-hound) [inline src/Tree/e-tree.scss]: Nesting should be no greater than 4, but was 5

File: src/Tree/e-tree.scss [hunk 6]
```diff
@@ -5,19 +5,26 @@
...
+      .e-tree-item-title { max-width: 180px; }
       .e-tree-item {
         padding: 0 0 0 60px; font-weight: normal; border-bottom: solid 1px #e0e0e0 !important;
         &:before { position: absolute; content: ''; width: 10px; height: 10px; box-sizing: border-box; border-bottom: solid 1px #e0e0e0; border-left: solid 1px #e0e0e0; left: 40px; top: 11px; }
       }
     }
-    .e-tree-list .e-tree-list .e-tree-item {
-      padding: 0 0 0 90px;
-      &:before { left: 70px; }
+    .e-tree-list .e-tree-list {
```
Comment (apress-hound) [inline src/Tree/e-tree.scss]: Merge rule `.e-tree-list .e-tree-list` with rule on line 7

File: src/Tree/e-tree.scss [hunk 7]
```diff
@@ -5,19 +5,26 @@
   &-list {
     margin: 0; padding: 0; width: 100%;
     .e-tree-list {
-      max-height: 0; transition: max-height .5s cubic-bezier(0.45, 0.05, 0.55, 0.95);
+      max-height: 0; transition: max-height .5s cubic-bezier(0.45, 0.05, 0.55, 0.95); overflow: hidden;
```
Comment (apress-hound) [inline src/Tree/e-tree.scss]: `0.45` should be written without a leading zero as `.45`<br>`0.05` should be written without a leading zero as `.05`<br>`0.55` should be written without a leading zero as `.55`<br>`0.95` should be written without a leading zero as `.95`

File: server.js [hunk 8]
```diff
@@ -48,7 +48,7 @@ app.get('/', function root(req, res) {
 
 app.post('/tree', (req, res) => {
   setTimeout(() => {
-    res.sendFile(__dirname + '/_mock/tree/tree2.json');
+    res.sendFile(__dirname + '/_mock/tree/tree.json');
```
Comment (apress-hound) [inline server.js]: '__dirname' is not defined.

File: dist/assets-help.bundle.js [hunk 9]
```diff
@@ -78,7 +78,7 @@ return /******/ (function(modules) { // webpackBootstrap
 /******/ 	}
 /******/ 	
 /******/ 	var hotApplyOnUpdate = true;
-/******/ 	var hotCurrentHash = "9f50f6153e2d5d87bf13"; // eslint-disable-line no-unused-vars
+/******/ 	var hotCurrentHash = "1bfe44bd5b8330cc9aac"; // eslint-disable-line no-unused-vars
```
Comment (apress-hound) [inline dist/assets-help.bundle.js]: 'hotCurrentHash' was used before it was defined.<br>Mixed double and single quotes.

**Comment by Kupov** [comment]
@dontuse

**Pull request merged**